src/fhir/input/fsh/aliases.fsh

```diff
@@ -2,3 +2,16 @@ Alias: $v2-0203 = http://terminology.hl7.org/CodeSystem/v2-0203
 Alias: $identifier-type-de-basis = http://fhir.de/CodeSystem/identifier-type-de-basis
 Alias: $data-absent-reason = http://hl7.org/fhir/StructureDefinition/data-absent-reason
 Alias: $v3-ActCode = http://terminology.hl7.org/CodeSystem/v3-ActCode
+// in Patient
+Alias: $identifier-kvid-10 = http://fhir.de/StructureDefinition/identifier-kvid-10
+Alias: $identifier-pkv = http://fhir.de/StructureDefinition/identifier-pkv
```

Comment:
PKV Identifier gibt es mit 1.4.0 nicht mehr oder?

Reply:
Abstimmung mit rundem FHIR-Tisch nötig, ob KVV-PKV-Identifier oder doch in de-basis verwendet wird.